CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
 # CURRENTLY-IN-DEVELOPMENT
+# `v0.14.0.1`
+### Controller Enhancements
+- VCSA Controllers
+    - 1230 - Add domain to input for each user in ldap identity source control - Feedback item #46
```

Comment:
The referenced PR numbers here are not accurate for this repo. Can you remove them please?